src/graphics/graphics-device.js

```diff
@@ -768,6 +768,8 @@ class GraphicsDevice extends EventHandler {
     destroy() {
         const gl = this.gl;
 
+        this.fire('destroy');
```

Comment:
is the cache handing this event? I'm not sure I see the code (in this PR)

Reply:
No it isn't. I thought I'd need it but didn't in the end. I meant to ask in the PR summary whether it might be worth keeping the event anyway to address device destroy in future. Also happy to remove until we need it. cc @willeastcott

Reply:
so how do textures get destroyed / removed from cache when the device is destroyed?

Reply:
they don't. I'll update it to clear the cache when device is destroyed.